seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/sink/KafkaSinkWriter.java

```diff
@@ -102,7 +107,8 @@ public KafkaSinkWriter(
 
     @Override
     public void write(SeaTunnelRow element) {
-        ProducerRecord<byte[], byte[]> producerRecord = seaTunnelRowSerializer.serializeRow(element);
+        String topic = topicExtractor.apply(element);
```

Comment:
I don't think `topicExtractor.apply(element)` called by each row is a good idea when `topic` is a fixed value. It means each row will make an unnecessary regular match.

So, I think you need improve the code and don't call `topicExtractor.apply(element)` if the `topic` is a fixed value.

Reply:
Good idea. Thanks for your advice.